src/main/java/org/uast/astgen/parser/LabelFactory.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * MIT License Copyright (c) 2022 unified-ast
+ * https://github.com/unified-ast/ast-generator/blob/master/LICENSE.txt
+ */
+package org.uast.astgen.parser;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * The label factory for label generation.
+ * The label cannot be an empty string and can be used as a variable name.
+ *
+ * @since 1.0
+ */
+public class LabelFactory {
+    /**
+     * The list of labels.
+     */
+    private static final List<String> LIST =
+        Arrays.asList(
```

Comment:
In my opinion, ordinal naming like `one, two, three ...` or `first, second, third ...` would be more user-friendly.

Reply:
Fixed